src/main/resources/templates/admin/moduleView.html

```diff
@@ -0,0 +1,535 @@
+                        <input type="hidden" name="id" th:value="${event.id}"/>
+                        <input type="submit"
+                               value="Update" />
+                    </form>
+                </td>
+                <td>
+                    <form method="POST"
+                          th:id="|delete-${event.id}|"
+                          th:action="|/admin/modules/${module.id}/deleteEvent|">
+                        <input type="hidden" name="id" th:value="${event.id}"/>
+                        <input type="submit"
+                               th:disabled="${event.projects.size() > 0}"
+                               value="Delete" />
+                    </form>
+                </td>
+            </tr>
+
+            <tr class="striped">
+                <td></td>
+                <td>
+                    <input type="datetime-local"
+                           name="startTime"
+                           required
+                           form="add-one-on-one-event" />
+                </td>
+                <td>
+                    <input type="datetime-local"
+                           name="endTime"
+                           required
+                           form="add-one-on-one-event" />
+                </td>
+                <td>
+                    <input type="url"
+                           name="meetingUrl"
+                           form="add-one-on-one-event" />
+                </td>
+                <td>
+                    <input type="url"
+                           name="recordingUrl"
+                           form="add-one-on-one-event" />
+                </td>
+                <td>
+                    <input type="url"
+                           name="slidesUrl"
+                           form="add-one-on-one-event" />
+                </td>
+                <td>
+                </td>
+                <td>
+                    <select name="toAdd"
+                            form="add-one-on-one-event"
+                            multiple>
+                        <option th:each="projectId : ${moduleProjects}"
+                                th:text="${cohortProjectNames.get(projectId)}"
+                                th:value="${projectId}">
+                            Project Name
+                        </option>
+                    </select>
+                </td>
+                <td>
+                    <form method="POST"
+                          id="add-one-on-one-event"
+                          th:action="|/admin/modules/${module.id}/createEvent|">
+                        <input type="hidden" name="eventTypeId" th:value="1"/>
+                        <input type="submit"
+                               value="Create" />
+                    </form>
+                </td>
+            </tr>
+            </tbody>
+        </table>
+    </th:block>
+
+    <th:block th:if="${workshopInfo}">
```

Comment:
Feels like a fair bit of duplication here. Could this maybe iterate over a list of event types and look each type's description and events up from the maps in the module model, rather than the server splitting them out into separate template variables?